resources/options_lookup.tsv

```diff
@@ -626,20 +626,20 @@ Insulation Wall	"Double Wood Stud, R-45, Grade 3"	ResidentialConstructionsWallsD
-Insulation Wall	"CMU, 6-in Hollow, R-11"	ResidentialConstructionsWallsCMU	thick_in=6	conductivity=4.29	density=65	framing_factor=0.076	furring_r=11	furring_cavity_depth_in=2	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"CMU, 6-in Hollow, R-15"	ResidentialConstructionsWallsCMU	thick_in=6	conductivity=4.29	density=65	framing_factor=0.076	furring_r=15	furring_cavity_depth_in=4	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"CMU, 6-in Hollow, R-19"	ResidentialConstructionsWallsCMU	thick_in=6	conductivity=4.29	density=65	framing_factor=0.076	furring_r=19	furring_cavity_depth_in=5.5	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"Brick, 12-in, 3-wythe, Uninsulated"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=0	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"Brick, 12-in, 3-wythe, R-7"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=7	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"Brick, 12-in, 3-wythe, R-11"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=11	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"Brick, 12-in, 3-wythe, R-15"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=15	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"Brick, 12-in, 3-wythe, R-19"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=19	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"CMU, 12-in Hollow"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=0	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"CMU, 12-in Hollow, R-10"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=10	furring_cavity_depth_in=2	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
+Insulation Wall	"CMU, 6-in Concrete Filled"	ResidentialConstructionsWallsCMU	thick_in=6	conductivity=5.33	density=140	framing_factor=0.076	furring_r=0	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				

```

Comment:
Using values from OS-HPXML for [brick](https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/materials.rb#L407) and [concrete](https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/materials.rb#L358) for density inputs.